### PR TITLE
ngram long format

### PR DESCRIPTION
**Pull request by double-y**
### Description
update ngram to output gathered format

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [x] Add test cases for this fix/enhancement
- [x] Pass devtools::check()
- [x] Pass devtools::test()
- [x] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/string_operation.R [hunk 1]
```diff
@@ -237,13 +237,13 @@ do_ngram <- function(df, token, sentence, document, maxn=2, sep="_"){
...
 
   # this is executed for ngrams not to be connected over sentences
-  grouped <- (df %>%
-            dplyr::group_by_(.dots=list(as.symbol(document_col), as.symbol(sentence_col)))) # convert the column name to symbol for colum names with backticks
+  grouped <- df %>%
+    dplyr::group_by_(.dots=list(as.symbol(document_col), as.symbol(sentence_col))) # convert the column name to symbol for colum names with backticks
   prev_cname <- token_col
   # create gram2, gram3, gram4, ... columns in this iteration
   for(n in seq(maxn)[-1]){
     # create a column name that doesn't conflict with the existing column names
```
Comment (hideaki) [inline R/string_operation.R]: Is this comment still relevant? Looks like avoid_conflict is removed.

File: R/string_operation.R [hunk 2]
```diff
@@ -263,7 +263,13 @@ do_ngram <- function(df, token, sentence, document, maxn=2, sep="_"){
     # preserve the cname to be used in next iteration
     prev_cname <- cname
   }
-  dplyr::ungroup(grouped)
+  ret <- dplyr::ungroup(grouped)
+
+  colnames(ret)[colnames(ret) == token_col] <- 1
```
Comment (hideaki) [inline R/string_operation.R]: Not too sure what this line is doing. Comment would help.

File: R/string_operation.R (hunk 1 above)
Comment (double-y) [inline R/string_operation.R]: modified the comments

File: R/string_operation.R (hunk 2 above)
Comment (double-y) [inline R/string_operation.R]: added a comment

**Pull request merged**